<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  






























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Research | Computational Genomics &amp; AI Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Research">
<meta name="description" content="at Inha University. ">

<meta property="og:title" content="Research">
<meta property="og:site_title" content="Computational Genomics &amp; AI Lab">
<meta property="og:description" content="at Inha University. ">
<meta property="og:url" content="https://compgenai.github.io">
<meta property="og:image" content="/images/share.jpg">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Research">
<meta property="twitter:description" content="at Inha University. ">
<meta property="twitter:url" content="https://compgenai.github.io">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="/images/share.jpg">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Research",
    "description": "at Inha University. ",
    "headline": "Research",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": "https://compgenai.github.io"
  }
</script>

<link rel="alternate" type="application/rss+xml" href="https://compgenai.github.io/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Barlow:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet" media="none" onload="this.removeAttribute('media'); this.onload = null;">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/details.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/table-wrap.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/background.jpg')">
  <a href="/" class="home">
    
      <span class="logo">
        
          <?xml version="1.0" encoding="UTF-8"?>
<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" version="1.1" viewbox="0 0 80 100">
  <!-- Generator: Adobe Illustrator 29.1.0, SVG Export Plug-In . SVG Version: 2.1.0 Build 142)  -->
  <defs>
    <style>
      .st0 {
        fill: none;
        stroke: #040000;
        stroke-miterlimit: 10;
      }

      .st1 {
        fill: #040000;
      }

      .st2 {
        fill: #2c2b30;
      }

      .st3 {
        fill: #1b75bc;
      }
    </style>
  </defs>
  <g>
    <line class="st0" x1="49.9" y1="77.7" x2="51.1" y2="67.1"></line>
    <line class="st0" x1="45.5" y1="45.8" x2="54.7" y2="35.9"></line>
    <line class="st0" x1="52" y1="22.2" x2="55.7" y2="29.3"></line>
    <line class="st0" x1="69.6" y1="28.7" x2="61.8" y2="31.7"></line>
    <line class="st0" x1="62.3" y1="54.2" x2="54.2" y2="61.2"></line>
    <line class="st0" x1="59.5" y1="37.1" x2="63.8" y2="46.7"></line>
    <line class="st0" x1="45.6" y1="21.7" x2="40.8" y2="33.3"></line>
    <line class="st0" x1="40.8" y1="67.1" x2="49.2" y2="84.6"></line>
    <line class="st0" x1="55.3" y1="19.1" x2="69.4" y2="24.5"></line>
    <line class="st0" x1="54.6" y1="81.4" x2="69.2" y2="75.2"></line>
    <line class="st0" x1="45" y1="55.3" x2="49.2" y2="60.9"></line>
    <line class="st0" x1="72.7" y1="32.1" x2="67.7" y2="46.7"></line>
    <line class="st0" x1="72.6" y1="67.3" x2="67.8" y2="55.8"></line>
  </g>
  <g>
    <path class="st1" d="M44.2,16.7c0-2.8,2.2-5,5-5s5,2.2,5,5-2.2,5-5,5-5-2.2-5-5Z"></path>
    <path class="st1" d="M44.2,83.7c0-2.8,2.2-5,5-5s5,2.2,5,5-2.2,5-5,5-5-2.2-5-5Z"></path>
    <path class="st1" d="M69.9,26.7c0-2.8,2.2-5,5-5s5,2.2,5,5-2.2,5-5,5-5-2.2-5-5Z"></path>
    <path class="st1" d="M69.9,72.7c0-2.8,2.2-5,5-5s5,2.2,5,5-2.2,5-5,5-5-2.2-5-5Z"></path>
    <circle class="st1" cx="65.8" cy="51.2" r="4.1"></circle>
    <circle class="st1" cx="57.7" cy="33.2" r="3.5"></circle>
    <circle class="st1" cx="51.5" cy="63.5" r="2.7"></circle>
  </g>
  <path class="st3" d="M36.4,50.5c0-2.9,2.4-5.3,5.3-5.3s5.3,2.4,5.3,5.3-2.4,5.3-5.3,5.3-5.3-2.4-5.3-5.3Z"></path>
  <path class="st3" d="M33.7,62.2c0-2.5,2-4.5,4.5-4.5s4.5,2,4.5,4.5-2,4.5-4.5,4.5-4.5-2-4.5-4.5Z"></path>
  <path class="st3" d="M33.7,38.2c0-2.5,2-4.5,4.5-4.5s4.5,2,4.5,4.5-2,4.5-4.5,4.5-4.5-2-4.5-4.5Z"></path>
  <path class="st3" d="M27.3,71.2c0-2.2,1.8-4.1,4.1-4.1s4.1,1.8,4.1,4.1-1.8,4.1-4.1,4.1-4.1-1.8-4.1-4.1Z"></path>
  <circle class="st3" cx="31" cy="29.6" r="4.1" transform="translate(-.9 .9) rotate(-1.7)"></circle>
  <path class="st2" d="M19.7,77.1c0-2,1.6-3.7,3.7-3.7s3.7,1.6,3.7,3.7-1.6,3.7-3.7,3.7-3.7-1.6-3.7-3.7Z"></path>
  <path class="st3" d="M19.7,23.7c0-2,1.6-3.7,3.7-3.7s3.7,1.6,3.7,3.7-1.6,3.7-3.7,3.7-3.7-1.6-3.7-3.7Z"></path>
  <path class="st3" d="M12.7,83.4c0-1.7,1.4-3.2,3.2-3.2s3.2,1.4,3.2,3.2-1.4,3.2-3.2,3.2-3.2-1.4-3.2-3.2Z"></path>
  <path class="st3" d="M12.7,17.1c0-1.7,1.4-3.2,3.2-3.2s3.2,1.4,3.2,3.2-1.4,3.2-3.2,3.2-3.2-1.4-3.2-3.2Z"></path>
  <path class="st3" d="M7,90.5c0-1.6,1.3-2.8,2.8-2.8s2.8,1.3,2.8,2.8-1.3,2.8-2.8,2.8-2.8-1.3-2.8-2.8Z"></path>
  <path class="st3" d="M7,10.2c0-1.6,1.3-2.8,2.8-2.8s2.8,1.3,2.8,2.8-1.3,2.8-2.8,2.8-2.8-1.3-2.8-2.8Z"></path>
  <path class="st3" d="M3.7,97.6c0-1.3,1.1-2.4,2.4-2.4s2.4,1.1,2.4,2.4-1.1,2.4-2.4,2.4-2.4-1.1-2.4-2.4Z"></path>
  <path class="st3" d="M3.7,2.4c0-1.3,1.1-2.4,2.4-2.4s2.4,1.1,2.4,2.4-1.1,2.4-2.4,2.4-2.4-1.1-2.4-2.4Z"></path>
  <path class="st2" d="M10.6,49.5c0,2.9-2.4,5.3-5.3,5.3s-5.3-2.4-5.3-5.3,2.4-5.3,5.3-5.3,5.3,2.4,5.3,5.3Z"></path>
  <path class="st2" d="M13.2,37.8c0,2.5-2,4.5-4.5,4.5s-4.5-2-4.5-4.5,2-4.5,4.5-4.5,4.5,2,4.5,4.5Z"></path>
  <path class="st2" d="M13.2,61.8c0,2.5-2,4.5-4.5,4.5s-4.5-2-4.5-4.5,2-4.5,4.5-4.5,4.5,2,4.5,4.5Z"></path>
  <path class="st2" d="M19.6,28.8c0,2.2-1.8,4.1-4.1,4.1s-4.1-1.8-4.1-4.1,1.8-4.1,4.1-4.1,4.1,1.8,4.1,4.1Z"></path>
  <circle class="st2" cx="15.9" cy="70.4" r="4.1"></circle>
  <circle class="st2" cx="31.1" cy="16.6" r="3.2" transform="translate(9.8 44.6) rotate(-80.8)"></circle>
  <circle class="st2" cx="31.1" cy="82.9" r="3.2" transform="translate(-55.7 100.3) rotate(-80.8)"></circle>
  <path class="st2" d="M39.9,9.5c0,1.6-1.3,2.8-2.8,2.8s-2.8-1.3-2.8-2.8,1.3-2.8,2.8-2.8,2.8,1.3,2.8,2.8Z"></path>
  <path class="st2" d="M39.9,89.8c0,1.6-1.3,2.8-2.8,2.8s-2.8-1.3-2.8-2.8,1.3-2.8,2.8-2.8,2.8,1.3,2.8,2.8Z"></path>
  <path class="st2" d="M43.2,2.4c0,1.3-1.1,2.4-2.4,2.4s-2.4-1.1-2.4-2.4,1.1-2.4,2.4-2.4,2.4,1.1,2.4,2.4Z"></path>
  <path class="st2" d="M43.2,97.6c0,1.3-1.1,2.4-2.4,2.4s-2.4-1.1-2.4-2.4,1.1-2.4,2.4-2.4,2.4,1.1,2.4,2.4Z"></path>
</svg>
        
      </span>
    
    
      <span class="title-text" data-tooltip="Home">
        
          <span class="title">Computational Genomics &amp; AI Lab</span>
        
        
          <span class="subtitle">at Inha University</span>
        
      </span>
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/research/" data-tooltip="Published works">
          Research
        </a>
      
    
      
        <a href="/projects/" data-tooltip="Software, datasets, and more">
          Projects
        </a>
      
    
      
        <a href="/team/" data-tooltip="About our team">
          Team
        </a>
      
    
      
        <a href="/blog/" data-tooltip="Musings and miscellany">
          Blog
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Email, address, and location">
          Contact
        </a>
      
    
  </nav>
</header>

    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - filter out blank sections
-->






  
  
  

  <section class="background" data-size="page">
    <h1 id="research">
<i class="icon fa-solid fa-microscope"></i>Research</h1>

<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="highlighted">Highlighted</h2>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/c7np" class="citation-image" aria-label="Open collaborative writing with Manubot">
        <img src="" alt="Open collaborative writing with Manubot" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/c7np" class="citation-title">
        Open collaborative writing with Manubot
      </a>

      <div class="citation-authors" tabindex="0">
        Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLOS Computational Biology</span>
         · 
        <span class="citation-date">24 Jun 2019</span>
         · 
        <span class="citation-id">doi:10.1371/journal.pcbi.1007128</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="all">All</h2>

<div class="search-box">
  <input type="text" class="search-input" oninput="onSearchInput(this)" placeholder="Search items on this page">
  <button disabled data-tooltip="Clear search" aria-label="clear search" onclick="onSearchClear()">
    <i class="icon fa-solid fa-magnifying-glass"></i>
  </button>
</div>

<div class="search-info"></div>

<h3 id="2025">2025</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9rfr3" class="citation-image" aria-label="Scalable data harmonization for single-cell image-based profiling with CytoTable">
        <img src="" alt="Scalable data harmonization for single-cell image-based profiling with CytoTable" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/g9rfr3" class="citation-title">
        Scalable data harmonization for single-cell image-based profiling with CytoTable
      </a>

      <div class="citation-authors" tabindex="0">
        Dave Bunten, Jenna Tomkinson, Erik Serrano, Michael J. Lippincott, Kenneth I. Brewer, Vince Rubinetti, Faisal Alquaddoomi, Gregory P. Way

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cold Spring Harbor Laboratory</span>
         · 
        <span class="citation-date">25 Jun 2025</span>
         · 
        <span class="citation-id">doi:10.1101/2025.06.19.660613</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2023">2023</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gs6kmr" class="citation-image" aria-label="The Monarch Initiative in 2024: an analytic platform integrating phenotypes, genes and diseases across species">
        <img src="" alt="The Monarch Initiative in 2024: an analytic platform integrating phenotypes, genes and diseases across species" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gs6kmr" class="citation-title">
        The Monarch Initiative in 2024: an analytic platform integrating phenotypes, genes and diseases across species
      </a>

      <div class="citation-authors" data-tooltip="Tim E Putman, Kevin Schaper, Nicolas Matentzoglu, Vincent P Rubinetti, Faisal S Alquaddoomi, Corey Cox, J Harry Caufield, Glass Elsarboukh, Sarah Gehrke, Harshad Hegde, Justin T Reese, Ian Braun, Richard M Bruskiewich, Luca Cappelletti, Seth Carbon, Anita R Caron, Lauren E Chan, Christopher G Chute, Katherina G Cortes, Vinícius De Souza, Tommaso Fontana, Nomi L Harris, Emily L Hartley, Eric Hurwitz, Julius O B Jacobsen, Madan Krishnamurthy, Bryan J Laraway, James A McLaughlin, Julie A McMurry, Sierra A T Moxon, Kathleen R Mullen, Shawn T O’Neil, Kent A Shefchek, Ray Stefancsik, Sabrina Toro, Nicole A Vasilevsky, Ramona L Walls, Patricia L Whetzel, David Osumi-Sutherland, Damian Smedley, Peter N Robinson, Christopher J Mungall, Melissa A Haendel, Monica C Munoz-Torres" tabindex="0">
        Tim E Putman, Kevin Schaper, Nicolas Matentzoglu, Vincent P Rubinetti, Faisal S Alquaddoomi, …, Damian Smedley, Peter N Robinson, Christopher J Mungall, Melissa A Haendel, Monica C Munoz-Torres

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nucleic Acids Research</span>
         · 
        <span class="citation-date">24 Nov 2023</span>
         · 
        <span class="citation-id">doi:10.1093/nar/gkad1082</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gsvf5z" class="citation-image" aria-label="Integration of 168,000 samples reveals global patterns of the human gut microbiome">
        <img src="" alt="Integration of 168,000 samples reveals global patterns of the human gut microbiome" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gsvf5z" class="citation-title">
        Integration of 168,000 samples reveals global patterns of the human gut microbiome
      </a>

      <div class="citation-authors" tabindex="0">
        Richard J. Abdill, Samantha P. Graham, Vincent Rubinetti, Frank W. Albert, Casey S. Greene, Sean Davis, Ran Blekhman

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cold Spring Harbor Laboratory</span>
         · 
        <span class="citation-date">11 Oct 2023</span>
         · 
        <span class="citation-id">doi:10.1101/2023.10.11.560955</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gr5hb5" class="citation-image" aria-label="MyGeneset.info: an interactive and programmatic platform for community-curated and user-created collections of genes">
        <img src="" alt="MyGeneset.info: an interactive and programmatic platform for community-curated and user-created collections of genes" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gr5hb5" class="citation-title">
        MyGeneset.info: an interactive and programmatic platform for community-curated and user-created collections of genes
      </a>

      <div class="citation-authors" tabindex="0">
        Ricardo Avila, Vincent Rubinetti, Xinghua Zhou, Dongbo Hu, Zhongchao Qian, Marco Alvarado Cano, Everaldo Rodolpho, Ginger Tsueng, Casey Greene, Chunlei Wu

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nucleic Acids Research</span>
         · 
        <span class="citation-date">18 Apr 2023</span>
         · 
        <span class="citation-id">doi:10.1093/nar/gkad289</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/grmcb9" class="citation-image" aria-label="Hetnet connectivity search provides rapid insights into how two biomedical entities are related">
        <img src="" alt="Hetnet connectivity search provides rapid insights into how two biomedical entities are related" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/grmcb9" class="citation-title">
        Hetnet connectivity search provides rapid insights into how two biomedical entities are related
      </a>

      <div class="citation-authors" data-tooltip="Daniel S. Himmelstein, Michael Zietz, Vincent Rubinetti, Kyle Kloster, Benjamin J. Heil, Faisal Alquaddoomi, Dongbo Hu, David N. Nicholson, Yun Hao, Blair D. Sullivan, Michael W. Nagle, Casey S. Greene" tabindex="0">
        Daniel S. Himmelstein, Michael Zietz, Vincent Rubinetti, Kyle Kloster, Benjamin J. Heil, …, David N. Nicholson, Yun Hao, Blair D. Sullivan, Michael W. Nagle, Casey S. Greene

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cold Spring Harbor Laboratory</span>
         · 
        <span class="citation-date">07 Jan 2023</span>
         · 
        <span class="citation-id">doi:10.1101/2023.01.05.522941</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2022">2022</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gsd85n" class="citation-image" aria-label="Hetnet connectivity search provides rapid insights into how biomedical entities are related">
        <img src="" alt="Hetnet connectivity search provides rapid insights into how biomedical entities are related" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gsd85n" class="citation-title">
        Hetnet connectivity search provides rapid insights into how biomedical entities are related
      </a>

      <div class="citation-authors" data-tooltip="Daniel S Himmelstein, Michael Zietz, Vincent Rubinetti, Kyle Kloster, Benjamin J Heil, Faisal Alquaddoomi, Dongbo Hu, David N Nicholson, Yun Hao, Blair D Sullivan, Michael W Nagle, Casey S Greene" tabindex="0">
        Daniel S Himmelstein, Michael Zietz, Vincent Rubinetti, Kyle Kloster, Benjamin J Heil, …, David N Nicholson, Yun Hao, Blair D Sullivan, Michael W Nagle, Casey S Greene

      </div>

      <div class="citation-details">
        <span class="citation-publisher">GigaScience</span>
         · 
        <span class="citation-date">28 Dec 2022</span>
         · 
        <span class="citation-id">doi:10.1093/gigascience/giad047</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gstx7j" class="citation-image" aria-label="MolEvolvR: A web-app for characterizing proteins using molecular evolution and phylogeny">
        <img src="" alt="MolEvolvR: A web-app for characterizing proteins using molecular evolution and phylogeny" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gstx7j" class="citation-title">
        MolEvolvR: A web-app for characterizing proteins using molecular evolution and phylogeny
      </a>

      <div class="citation-authors" data-tooltip="Jacob D Krol, Joseph T Burke, Samuel Z Chen, Lo M Sosinski, Faisal S Alquaddoomi, Evan P Brenner, Ethan P Wolfe, Vincent P Rubinetti, Shaddai Amolitos, Kellen M Reason, John B Johnston, Janani Ravi" tabindex="0">
        Jacob D Krol, Joseph T Burke, Samuel Z Chen, Lo M Sosinski, Faisal S Alquaddoomi, …, Vincent P Rubinetti, Shaddai Amolitos, Kellen M Reason, John B Johnston, Janani Ravi

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cold Spring Harbor Laboratory</span>
         · 
        <span class="citation-date">22 Feb 2022</span>
         · 
        <span class="citation-id">doi:10.1101/2022.02.18.461833</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2020">2020</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gg2mjh" class="citation-image" aria-label="Compressing gene expression data using multiple latent space dimensionalities learns complementary biological representations">
        <img src="" alt="Compressing gene expression data using multiple latent space dimensionalities learns complementary biological representations" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gg2mjh" class="citation-title">
        Compressing gene expression data using multiple latent space dimensionalities learns complementary biological representations
      </a>

      <div class="citation-authors" tabindex="0">
        Gregory P. Way, Michael Zietz, Vincent Rubinetti, Daniel S. Himmelstein, Casey S. Greene

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Genome Biology</span>
         · 
        <span class="citation-date">11 May 2020</span>
         · 
        <span class="citation-id">doi:10.1186/s13059-020-02021-3</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2019">2019</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/c7np" class="citation-image" aria-label="Open collaborative writing with Manubot">
        <img src="" alt="Open collaborative writing with Manubot" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/c7np" class="citation-title">
        Open collaborative writing with Manubot
      </a>

      <div class="citation-authors" tabindex="0">
        Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLOS Computational Biology</span>
         · 
        <span class="citation-date">24 Jun 2019</span>
         · 
        <span class="citation-id">doi:10.1371/journal.pcbi.1007128</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gfxjxf" class="citation-image" aria-label="Sequential compression of gene expression across dimensionalities and methods reveals no single best method or dimensionality">
        <img src="" alt="Sequential compression of gene expression across dimensionalities and methods reveals no single best method or dimensionality" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      

      <a href="https://doi.org/gfxjxf" class="citation-title">
        Sequential compression of gene expression across dimensionalities and methods reveals no single best method or dimensionality
      </a>

      <div class="citation-authors" tabindex="0">
        Gregory P. Way, Michael Zietz, Vincent Rubinetti, Daniel S. Himmelstein, Casey S. Greene

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cold Spring Harbor Laboratory</span>
         · 
        <span class="citation-date">11 Mar 2019</span>
         · 
        <span class="citation-id">doi:10.1101/573782</span>
      </div>

      
        

        

        
      
    </div>
  </div>
</div>

<h3></h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/fkmgt8" class="citation-image" aria-label="ConPlex: a server for the evolutionary conservation analysis of protein complex structures">
        <img src="" alt="ConPlex: a server for the evolutionary conservation analysis of protein complex structures" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/fkmgt8" class="citation-title">
        ConPlex: a server for the evolutionary conservation analysis of protein complex structures
      </a>

      <div class="citation-authors" tabindex="0">
        Y. S. Choi, S. K. Han, J. Kim, J.-S. Yang, J. Jeon, S. H. Ryu, S. Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nucleic Acids Research</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1093/nar/gkq328</span>
      </div>

      
        
          <div class="citation-description">
            This paper introduces ConPlex, a web server for evolutionary conservation analysis of protein complex structures. It identifies protein interfaces, evaluates residue-specific conservation, and visualizes conserved regions on 3D structures, providing a convenient tool for studying protein–protein interactions and distinguishing biologically relevant interfaces.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://academic.oup.com/nar/article/38/suppl_2/W450/1101081" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/dfc994" class="citation-image" aria-label="OASIS: Online Application for the Survival Analysis of Lifespan Assays Performed in Aging Research">
        <img src="" alt="OASIS: Online Application for the Survival Analysis of Lifespan Assays Performed in Aging Research" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/dfc994" class="citation-title">
        OASIS: Online Application for the Survival Analysis of Lifespan Assays Performed in Aging Research
      </a>

      <div class="citation-authors" tabindex="0">
        Jae-Seong Yang, Hyun-Jun Nam, Mihwa Seo, Seong Kyu Han, Yonghwan Choi, Hong Gil Nam, Seung-Jae Lee, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLoS ONE</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1371/journal.pone.0023525</span>
      </div>

      
        
          <div class="citation-description">
            This paper presents OASIS, an online application for survival analysis tailored to aging research. It provides Kaplan–Meier estimates, statistical tests, and visualizations such as survival and hazard plots, enabling efficient interpretation of lifespan assays and comparative survival data.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0023525" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f4ddjf" class="citation-image" aria-label="Structural differences between thermophilic and mesophilic membrane proteins">
        <img src="" alt="Structural differences between thermophilic and mesophilic membrane proteins" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f4ddjf" class="citation-title">
        Structural differences between thermophilic and mesophilic membrane proteins
      </a>

      <div class="citation-authors" tabindex="0">
        Alejandro D. Meruelo, Seong Kyu Han, Sanguk Kim, James U. Bowie

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Protein Science</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1002/pro.2157</span>
      </div>

      
        
          <div class="citation-description">
            This study compares structural features of mesophilic and thermophilic membrane proteins, revealing increased hydrophobicity of transmembrane helices in thermophiles. The analysis highlights evolutionary pressures to reduce polar and thermally sensitive residues, enhancing membrane partitioning and stability at high temperatures.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.2157" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gbcbzd" class="citation-image" aria-label="Network rewiring is an important mechanism of gene essentiality change">
        <img src="" alt="Network rewiring is an important mechanism of gene essentiality change" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gbcbzd" class="citation-title">
        Network rewiring is an important mechanism of gene essentiality change
      </a>

      <div class="citation-authors" tabindex="0">
        Jinho Kim, Inhae Kim, Seong Kyu Han, James U. Bowie, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Scientific Reports</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/srep00900</span>
      </div>

      
        
          <div class="citation-description">
            This study demonstrates that network rewiring is a key mechanism driving changes in gene essentiality between species. Comparative analysis of yeast and mouse shows that nonessential yeast genes become essential in mouse through increased network connectivity in protein complexes, reinforcing the centrality-lethality rule in evolutionary contexts.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/srep00900#citeas" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f4rt86" class="citation-image" aria-label="Rampant Exchange of the Structure and Function of Extramembrane Domains between Membrane and Water Soluble Proteins">
        <img src="" alt="Rampant Exchange of the Structure and Function of Extramembrane Domains between Membrane and Water Soluble Proteins" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f4rt86" class="citation-title">
        Rampant Exchange of the Structure and Function of Extramembrane Domains between Membrane and Water Soluble Proteins
      </a>

      <div class="citation-authors" tabindex="0">
        Hyun-Jun Nam, Seong Kyu Han, James U. Bowie, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLoS Computational Biology</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1371/journal.pcbi.1002997</span>
      </div>

      
        
          <div class="citation-description">
            This study shows that extramembrane domains are frequently exchanged between membrane and soluble proteins, with 67% of soluble domains structurally resembling known soluble proteins and 41% of soluble proteins sharing domains with membrane proteins. These exchanges enhance functional complexity, particularly in eukaryotes, and provide resources for predicting membrane protein function.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002997" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f5k5pm" class="citation-image" aria-label="Protein design by fusion: implications for protein structure prediction and evolution">
        <img src="" alt="Protein design by fusion: implications for protein structure prediction and evolution" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f5k5pm" class="citation-title">
        Protein design by fusion: implications for protein structure prediction and evolution
      </a>

      <div class="citation-authors" tabindex="0">
        Katarzyna Skorupka, Seong Kyu Han, Hyun-Jun Nam, Sanguk Kim, Salem Faham

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Acta Crystallographica Section D Biological Crystallography</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1107/S0907444913022701</span>
      </div>

      
        
          <div class="citation-description">
            This study reports the structure of a fusion protein combining the heterodimeric flagella-assembly proteins FliS and FliC, showing discrepancies with threading-based structural predictions. By comparing with naturally occurring homologous fusions, it highlights how domain fusion informs protein evolution and can improve structure prediction methods.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.iucr.org/paper?cnor=yt5059&buy=yes" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9384g" class="citation-image" aria-label="Linear Motif-Mediated Interactions Have Contributed to the Evolution of Modularity in Complex Protein Interaction Networks">
        <img src="" alt="Linear Motif-Mediated Interactions Have Contributed to the Evolution of Modularity in Complex Protein Interaction Networks" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g9384g" class="citation-title">
        Linear Motif-Mediated Interactions Have Contributed to the Evolution of Modularity in Complex Protein Interaction Networks
      </a>

      <div class="citation-authors" tabindex="0">
        Inhae Kim, Heetak Lee, Seong Kyu Han, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLoS Computational Biology</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1371/journal.pcbi.1003881</span>
      </div>

      
        
          <div class="citation-description">
            This study shows that weak domain–linear motif interactions (DLIs) preferentially connect different biological modules in protein–protein interaction networks, in contrast to strong domain–domain interactions. DLIs act as evolutionary switches that facilitate modularity, rewiring, and functional innovation in complex eukaryotic networks.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003881" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f7mpkc" class="citation-image" aria-label="RNA helicase HEL-1 promotes longevity by specifically activating DAF-16 FOXO transcription factor signaling in i Caenorhabditis elegans i">
        <img src="" alt="RNA helicase HEL-1 promotes longevity by specifically activating DAF-16 FOXO transcription factor signaling in i Caenorhabditis elegans i" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f7mpkc" class="citation-title">
        RNA helicase HEL-1 promotes longevity by specifically activating DAF-16/FOXO transcription factor signaling in
            <i>Caenorhabditis elegans</i>
      </a>

      <div class="citation-authors" data-tooltip="Mihwa Seo, Keunhee Seo, Wooseon Hwang, Hee Jung Koo, Jeong-Hoon Hahm, Jae-Seong Yang, Seong Kyu Han, Daehee Hwang, Sanguk Kim, Sung Key Jang, Yoontae Lee, Hong Gil Nam, Seung-Jae V. Lee" tabindex="0">
        Mihwa Seo, Keunhee Seo, Wooseon Hwang, Hee Jung Koo, Jeong-Hoon Hahm, …, Sanguk Kim, Sung Key Jang, Yoontae Lee, Hong Gil Nam, Seung-Jae V. Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Proceedings of the National Academy of Sciences</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1073/pnas.1505451112</span>
      </div>

      
        
          <div class="citation-description">
            This study identifies the RNA helicase HEL-1 as a regulator of longevity in C. elegans by specifically activating the DAF-16/FOXO signaling pathway. HEL-1 is essential for lifespan extension under reduced insulin/IGF-1 signaling and functions as a co-regulator of DAF-16 target genes, suggesting conserved mechanisms for RNA helicases in mammalian aging.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.pnas.org/doi/abs/10.1073/pnas.1505451112" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f727dh" class="citation-image" aria-label="Network Modules of the Cross-Species Genotype-Phenotype Map Reflect the Clinical Severity of Human Diseases">
        <img src="" alt="Network Modules of the Cross-Species Genotype-Phenotype Map Reflect the Clinical Severity of Human Diseases" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f727dh" class="citation-title">
        Network Modules of the Cross-Species Genotype-Phenotype Map Reflect the Clinical Severity of Human Diseases
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Inhae Kim, Jihye Hwang, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">PLOS ONE</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1371/journal.pone.0136300</span>
      </div>

      
        
          <div class="citation-description">
            This study constructs a cross-species genotype–phenotype map to explain clinical severity in human diseases. By integrating essential gene data from model organisms, the work shows that diseases enriched with essential genes exhibit higher mortality, multiple comorbidities, and pleiotropy, providing insights into disease progression and gene–disease associations.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0136300" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f8vmbg" class="citation-image" aria-label="Spread of Mutant Middle East Respiratory Syndrome Coronavirus with Reduced Affinity to Human CD26 during the South Korean Outbreak">
        <img src="" alt="Spread of Mutant Middle East Respiratory Syndrome Coronavirus with Reduced Affinity to Human CD26 during the South Korean Outbreak" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f8vmbg" class="citation-title">
        Spread of Mutant Middle East Respiratory Syndrome Coronavirus with Reduced Affinity to Human CD26 during the South Korean Outbreak
      </a>

      <div class="citation-authors" data-tooltip="Yuri Kim, Shinhye Cheon, Chan-Ki Min, Kyung Mok Sohn, Ying Jin Kang, Young-Je Cha, Ju-Il Kang, Seong Kyu Han, Na-Young Ha, Gwanghun Kim, Abdimadiyeva Aigerim, Hyun Mu Shin, Myung-Sik Choi, Sanguk Kim, Hyun-Soo Cho, Yeon-Sook Kim, Nam-Hyuk Cho" tabindex="0">
        Yuri Kim, Shinhye Cheon, Chan-Ki Min, Kyung Mok Sohn, Ying Jin Kang, …, Myung-Sik Choi, Sanguk Kim, Hyun-Soo Cho, Yeon-Sook Kim, Nam-Hyuk Cho

      </div>

      <div class="citation-details">
        <span class="citation-publisher">mBio</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1128/mbio.00019-16</span>
      </div>

      
        
          <div class="citation-description">
            This study reports the emergence of mutant MERS-CoV strains during the 2015 South Korean outbreak with reduced affinity to the human CD26 receptor. Genomic and functional analyses reveal that mutations in the spike protein receptor-binding domain impair viral entry and fitness, suggesting adaptation driven by host immune pressure rather than enhanced receptor binding.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://journals.asm.org/doi/full/10.1128/mbio.00019-16" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f9qbpq" class="citation-image" aria-label="OASIS 2: online application for survival analysis 2 with features for the analysis of maximal lifespan and healthspan in aging research">
        <img src="" alt="OASIS 2: online application for survival analysis 2 with features for the analysis of maximal lifespan and healthspan in aging research" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f9qbpq" class="citation-title">
        OASIS 2: online application for survival analysis 2 with features for the analysis of maximal lifespan and healthspan in aging research
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Dongyeop Lee, Heetak Lee, Donghyo Kim, Heehwa G. Son, Jae-Seong Yang, Seung-Jae V. Lee, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Oncotarget</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.18632/oncotarget.11269</span>
      </div>

      
        
          <div class="citation-description">
            This paper introduces OASIS 2, an updated online platform for survival analysis with enhanced statistical tools and user interface. It supports maximal lifespan and healthspan analysis, enabling comparison of age-associated physiological changes, and serves as a comprehensive resource for aging research.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.oncotarget.com/article/11269/text/" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/f9hp7h" class="citation-image" aria-label="Constitutive activation of T cells by γ2-herpesviral GPCR through the interaction with cellular CXCR4">
        <img src="" alt="Constitutive activation of T cells by γ2-herpesviral GPCR through the interaction with cellular CXCR4" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/f9hp7h" class="citation-title">
        Constitutive activation of T cells by γ2-herpesviral GPCR through the interaction with cellular CXCR4
      </a>

      <div class="citation-authors" data-tooltip="Eun-Kyung Kwon, Chan-Ki Min, Yuri Kim, Jae-Won Lee, Abdimadiyeva Aigerim, Sebastian Schmidt, Hyun-Jun Nam, Seong Kyu Han, Kuglae Kim, Jeong Seok Cha, Hoyoung Kim, Sanguk Kim, Hyun-Soo Cho, Myung-Sik Choi, Nam-Hyuk Cho" tabindex="0">
        Eun-Kyung Kwon, Chan-Ki Min, Yuri Kim, Jae-Won Lee, Abdimadiyeva Aigerim, …, Hoyoung Kim, Sanguk Kim, Hyun-Soo Cho, Myung-Sik Choi, Nam-Hyuk Cho

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Biochimica et Biophysica Acta (BBA) - Molecular Cell Research</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.bbamcr.2016.10.008</span>
      </div>

      
        
          <div class="citation-description">
            This study shows that the γ2-herpesviral GPCR from Herpesvirus saimiri constitutively activates T cells through heteromeric interaction with cellular CXCR4. The mechanism is independent of proximal TCR signaling, highlighting how viral GPCR–CXCR4 cooperativity underlies persistent signaling and contributes to herpesviral pathogenesis.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.sciencedirect.com/science/article/pii/S0167488916302622" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gdd4g2" class="citation-image" aria-label="Divergence of Noncoding Regulatory Elements Explains Gene Phenotype Differences between Human and Mouse Orthologous Genes">
        <img src="" alt="Divergence of Noncoding Regulatory Elements Explains Gene Phenotype Differences between Human and Mouse Orthologous Genes" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gdd4g2" class="citation-title">
        Divergence of Noncoding Regulatory Elements Explains Gene–Phenotype Differences between Human and Mouse Orthologous Genes
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Donghyo Kim, Heetak Lee, Inhae Kim, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Molecular Biology and Evolution</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1093/molbev/msy056</span>
      </div>

      
        
          <div class="citation-description">
            This study demonstrates that gene–phenotype differences between human and mouse orthologs are largely explained by divergence in noncoding regulatory elements and tissue-specific expression, rather than protein-coding changes. Using semantic similarity and transcriptomic analysis, it emphasizes evolutionary context for accurate interpretation of model organism data.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://academic.oup.com/mbe/article/35/7/1653/4956777" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9384d" class="citation-image" aria-label="Exomic and transcriptomic alterations of hereditary gingival fibromatosis">
        <img src="/images/papers/10.1111_odi.13093_image.jpg" alt="Exomic and transcriptomic alterations of hereditary gingival fibromatosis" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g9384d" class="citation-title">
        Exomic and transcriptomic alterations of hereditary gingival fibromatosis
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Jungho Kong, Sanguk Kim, Jae‐Hoon Lee, Dong‐Hoo Han

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Oral Diseases</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1111/odi.13093</span>
      </div>

      
        
          <div class="citation-description">
            This study integrates exome sequencing and RNA-seq to uncover the genetic basis of hereditary gingival fibromatosis (HGF). By connecting exomic and transcriptomic alterations through regulatory network analysis, it highlights TGF-β/SMAD signaling and craniofacial development pathways, providing insight into fibroblast overgrowth mechanisms in HGF.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://onlinelibrary.wiley.com/doi/full/10.1111/odi.13093" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9384c" class="citation-image" aria-label="Link clustering explains non-central and contextually essential genes in protein interaction networks">
        <img src="/images/papers/10.1038_s41598-019-48273-3_image.jpg" alt="Link clustering explains non-central and contextually essential genes in protein interaction networks" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g9384c" class="citation-title">
        Link clustering explains non-central and contextually essential genes in protein interaction networks
      </a>

      <div class="citation-authors" tabindex="0">
        Inhae Kim, Heetak Lee, Kwanghwan Lee, Seong Kyu Han, Donghyo Kim, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Scientific Reports</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/s41598-019-48273-3</span>
      </div>

      
        
          <div class="citation-description">
            This study proposes link clustering as a network-based measure to identify contextually essential genes (EGs) in protein-protein interaction networks. The analysis shows that many non-central EGs, often overlooked by centrality measures, display variable essentiality across conditions, linking local network structure to functional importance.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/s41598-019-48273-3#Abs1" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9384b" class="citation-image" aria-label="Evolutionary coupling analysis identifies the impact of disease-associated variants at less-conserved sites">
        <img src="" alt="Evolutionary coupling analysis identifies the impact of disease-associated variants at less-conserved sites" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g9384b" class="citation-title">
        Evolutionary coupling analysis identifies the impact of disease-associated variants at less-conserved sites
      </a>

      <div class="citation-authors" tabindex="0">
        Donghyo Kim, Seong Kyu Han, Kwanghwan Lee, Inhae Kim, JungHo Kong, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nucleic Acids Research</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1093/nar/gkz536</span>
      </div>

      
        
          <div class="citation-description">
            This study introduces an evolutionary coupling analysis method to identify disease-associated variants (DVs) at less-conserved sites. By capturing compensatory interactions at functionally important positions, the approach reveals impactful variants missed by conservation-based methods, particularly at protein interaction interfaces with species-specific adaptations.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://academic.oup.com/nar/article/47/16/e94/5519167" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93839" class="citation-image" aria-label="Variants at potential loci associated with Sjogren s syndrome in Koreans: A genetic association study">
        <img src="" alt="Variants at potential loci associated with Sjogren s syndrome in Koreans: A genetic association study" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93839" class="citation-title">
        Variants at potential loci associated with Sjogren's syndrome in Koreans: A genetic association study
      </a>

      <div class="citation-authors" tabindex="0">
        Qingxia Shen, Kwanghwan Lee, Seong Kyu Han, Hyung-Joon Ahn, Sanguk Kim, Jae Hoon Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Clinical Immunology</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.clim.2019.07.010</span>
      </div>

      
        
          <div class="citation-description">
            This genetic association study in Korean patients with Sjögren’s syndrome identifies novel susceptibility loci, including PRAMEF13, TARBP1, and UGT2B28, in addition to confirming known genes such as MSH5 and RELN. The results provide new insights into immune-related pathways contributing to Sjögren’s syndrome in the Korean population.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.sciencedirect.com/science/article/pii/S1521661619302657" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93838" class="citation-image" aria-label="Domain-mediated interactions for protein subfamily identification">
        <img src="" alt="Domain-mediated interactions for protein subfamily identification" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93838" class="citation-title">
        Domain-mediated interactions for protein subfamily identification
      </a>

      <div class="citation-authors" tabindex="0">
        Heetak Lee, Inhae Kim, Seong Kyu Han, Donghyo Kim, Jungho Kong, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Scientific Reports</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/s41598-019-57187-z</span>
      </div>

      
        
          <div class="citation-description">
            This work introduces domain-mediated interactions (DMIs) as a basis for defining protein subfamilies within larger families. By linking shared domains with interaction partners, the study shows that DMI subfamilies correlate with disease associations in humans and phenotypic outcomes in yeast, offering a network-based framework for functional protein annotation.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/s41598-019-57187-z" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gh53qt" class="citation-image" aria-label="Single-cell RNA sequencing identifies shared differentiation paths of mouse thymic innate T cells">
        <img src="" alt="Single-cell RNA sequencing identifies shared differentiation paths of mouse thymic innate T cells" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gh53qt" class="citation-title">
        Single-cell RNA sequencing identifies shared differentiation paths of mouse thymic innate T cells
      </a>

      <div class="citation-authors" data-tooltip="Minji Lee, Eunmin Lee, Seong Kyu Han, Yoon Ha Choi, Dong-il Kwon, Hyobeen Choi, Kwanghwan Lee, Eun Seo Park, Min-Seok Rha, Dong Jin Joo, Eui-Cheol Shin, Sanguk Kim, Jong Kyoung Kim, You Jeong Lee" tabindex="0">
        Minji Lee, Eunmin Lee, Seong Kyu Han, Yoon Ha Choi, Dong-il Kwon, …, Dong Jin Joo, Eui-Cheol Shin, Sanguk Kim, Jong Kyoung Kim, You Jeong Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Communications</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/s41467-020-18155-8</span>
      </div>

      
        
          <div class="citation-description">
            This study uses single-cell RNA sequencing to reveal that iNKT, MAIT, and γδ T cells undergo shared, branched differentiation pathways in the thymus. Clonal selection accompanies their maturation, and functional experiments confirm transcription factor dependencies, highlighting conserved effector differentiation processes in both mice and humans.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/s41467-020-18155-8" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gh33cn" class="citation-image" aria-label="Network-based machine learning in colorectal and bladder organoid models predicts anti-cancer drug efficacy in patients">
        <img src="" alt="Network-based machine learning in colorectal and bladder organoid models predicts anti-cancer drug efficacy in patients" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gh33cn" class="citation-title">
        Network-based machine learning in colorectal and bladder organoid models predicts anti-cancer drug efficacy in patients
      </a>

      <div class="citation-authors" tabindex="0">
        JungHo Kong, Heetak Lee, Donghyo Kim, Seong Kyu Han, Doyeon Ha, Kunyoo Shin, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Communications</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/s41467-020-19313-8</span>
      </div>

      
        
          <div class="citation-description">
            This study introduces a network-based machine learning framework that leverages pharmacogenomic data from 3D organoid models to predict anti-cancer drug efficacy. The approach identifies robust transcriptomic biomarkers that accurately forecast patient responses to 5-fluorouracil in colorectal cancer and cisplatin in bladder cancer, validated across independent datasets.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/s41467-020-19313-8" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93837" class="citation-image" aria-label="Phylogenetic analysis of scp ABCG scp subfamily proteins in plants: functional clustering and coevolution with scp ABCGs scp of pathogens">
        <img src="" alt="Phylogenetic analysis of scp ABCG scp subfamily proteins in plants: functional clustering and coevolution with scp ABCGs scp of pathogens" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93837" class="citation-title">
        Phylogenetic analysis of <scp>ABCG</scp> subfamily proteins in plants: functional clustering and coevolution with <scp>ABCGs</scp> of pathogens
      </a>

      <div class="citation-authors" data-tooltip="Chung Hyun Cho, Sunghoon Jang, Bae Young Choi, Daewoong Hong, Du Seok Choi, Sera Choi, Haseong Kim, Seong Kyu Han, Sanguk Kim, Min‐Sung Kim, Michael Palmgren, Kee Hoon Sohn, Hwan Su Yoon, Youngsook Lee" tabindex="0">
        Chung Hyun Cho, Sunghoon Jang, Bae Young Choi, Daewoong Hong, Du Seok Choi, …, Min‐Sung Kim, Michael Palmgren, Kee Hoon Sohn, Hwan Su Yoon, Youngsook Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Physiologia Plantarum</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1111/ppl.13052</span>
      </div>

      
        
          <div class="citation-description">
            This phylogenetic study reveals massive duplication of ABCG subfamily proteins in both land plants and oomycete pathogens, suggesting coevolution. Functional evidence shows that plant ABCGs, such as AtABCG31, contribute to defense against pathogens, highlighting an evolutionary arms race involving secondary metabolite transport.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://onlinelibrary.wiley.com/doi/full/10.1111/ppl.13052" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gnpnx6" class="citation-image" aria-label="MON-2, a Golgi protein, mediates autophagy-dependent longevity in i Caenorhabditis elegans i">
        <img src="" alt="MON-2, a Golgi protein, mediates autophagy-dependent longevity in i Caenorhabditis elegans i" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gnpnx6" class="citation-title">
        MON-2, a Golgi protein, mediates autophagy-dependent longevity in
            <i>Caenorhabditis elegans</i>
      </a>

      <div class="citation-authors" data-tooltip="Yoonji Jung, Murat Artan, Nari Kim, Jeonghun Yeom, Ara B. Hwang, Dae-Eun Jeong, Özlem Altintas, Keunhee Seo, Mihwa Seo, Dongyeop Lee, Wooseon Hwang, Yujin Lee, Jooyeon Sohn, Eun Ji E. Kim, Sungeun Ju, Seong Kyu Han, Hyun-Jun Nam, Linnea Adams, Youngjae Ryu, Dong Jin Moon, Chanhee Kang, Joo-Yeon Yoo, Sang Ki Park, Chang Man Ha, Malene Hansen, Sanguk Kim, Cheolju Lee, Seung-Yeol Park, Seung-Jae V. Lee" tabindex="0">
        Yoonji Jung, Murat Artan, Nari Kim, Jeonghun Yeom, Ara B. Hwang, …, Malene Hansen, Sanguk Kim, Cheolju Lee, Seung-Yeol Park, Seung-Jae V. Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Science Advances</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1126/sciadv.abj8156</span>
      </div>

      
        
          <div class="citation-description">
            This study identifies MON-2, a Golgi protein, as a key mediator of autophagy-dependent longevity in C. elegans. MON-2, along with DOP1/PAD-1, promotes trafficking between the Golgi and endosome and activates the autophagy protein GABARAP/LGG-1, with conserved roles observed in mammalian cells linking Golgi function to lifespan regulation.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.science.org/doi/full/10.1126/sciadv.abj8156" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gtsx5v" class="citation-image" aria-label="Evolutionary rewiring of regulatory networks contributes to phenotypic differences between human and mouse orthologous genes">
        <img src="" alt="Evolutionary rewiring of regulatory networks contributes to phenotypic differences between human and mouse orthologous genes" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gtsx5v" class="citation-title">
        Evolutionary rewiring of regulatory networks contributes to phenotypic differences between human and mouse orthologous genes
      </a>

      <div class="citation-authors" tabindex="0">
        Doyeon Ha, Donghyo Kim, Inhae Kim, Youngchul Oh, JungHo Kong, Seong Kyu Han, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nucleic Acids Research</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1093/nar/gkac050</span>
      </div>

      
        
          <div class="citation-description">
            This study systematically investigates the evolutionary divergence of transcription factor–target gene networks between humans and mice. It shows that regulatory network rewiring contributes to phenotypic differences of orthologous genes, emphasizing the need to consider species-specific regulation when interpreting mouse models of human disease.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://academic.oup.com/nar/article/50/4/1849/6523801" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93836" class="citation-image" aria-label="Deconvolution of bulk tumors into distinct immune cell states predicts colorectal cancer recurrence">
        <img src="" alt="Deconvolution of bulk tumors into distinct immune cell states predicts colorectal cancer recurrence" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93836" class="citation-title">
        Deconvolution of bulk tumors into distinct immune cell states predicts colorectal cancer recurrence
      </a>

      <div class="citation-authors" data-tooltip="Donghyo Kim, Jinho Kim, Juhun Lee, Seong Kyu Han, Kwanghwan Lee, JungHo Kong, Yeon Jeong Kim, Woo Yong Lee, Seong Hyeon Yun, Hee Cheol Kim, Hye Kyung Hong, Yong Beom Cho, Donghyun Park, Sanguk Kim" tabindex="0">
        Donghyo Kim, Jinho Kim, Juhun Lee, Seong Kyu Han, Kwanghwan Lee, …, Hee Cheol Kim, Hye Kyung Hong, Yong Beom Cho, Donghyun Park, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">iScience</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.isci.2022.105392</span>
      </div>

      
        
          <div class="citation-description">
            This study shows that deconvolving bulk colorectal tumors into two immune cell states—cancer-associated and noncancer-associated—improves recurrence prediction. The approach outperforms traditional clinical and molecular predictors, underscoring the importance of distinguishing immune cell states to better explain recurrence risk.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.cell.com/iscience/fulltext/S2589-0042(22)01664-9" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/gszk5n" class="citation-image" aria-label="Quality assessment and refinement of chromatin accessibility data using a sequence-based predictive model">
        <img src="" alt="Quality assessment and refinement of chromatin accessibility data using a sequence-based predictive model" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/gszk5n" class="citation-title">
        Quality assessment and refinement of chromatin accessibility data using a sequence-based predictive model
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Yoshiharu Muto, Parker C. Wilson, Benjamin D. Humphreys, Matthew G. Sampson, Aravinda Chakravarti, Dongwon Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Proceedings of the National Academy of Sciences</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1073/pnas.2212810119</span>
      </div>

      
        
          <div class="citation-description">
            This work introduces gkmQC, a sequence-based machine learning framework for assessing and refining chromatin accessibility data quality. gkmQC effectively identifies high-quality samples, optimizes peak-calling thresholds, and improves the detection of functional regulatory elements, particularly enhancing insights for rare cell types in single-cell data.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.pnas.org/doi/abs/10.1073/pnas.2212810119" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93835" class="citation-image" aria-label="Mapping genomic regulation of kidney disease and traits through high-resolution and interpretable eQTLs">
        <img src="" alt="Mapping genomic regulation of kidney disease and traits through high-resolution and interpretable eQTLs" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93835" class="citation-title">
        Mapping genomic regulation of kidney disease and traits through high-resolution and interpretable eQTLs
      </a>

      <div class="citation-authors" data-tooltip="Seong Kyu Han, Michelle T. McNulty, Christopher J. Benway, Pei Wen, Anya Greenberg, Ana C. Onuchic-Whitford, Dongkeun Jang, Jason Flannick, Noël P. Burtt, Parker C. Wilson, Benjamin D. Humphreys, Xiaoquan Wen, Zhe Han, Dongwon Lee, Matthew G. Sampson" tabindex="0">
        Seong Kyu Han, Michelle T. McNulty, Christopher J. Benway, Pei Wen, Anya Greenberg, …, Benjamin D. Humphreys, Xiaoquan Wen, Zhe Han, Dongwon Lee, Matthew G. Sampson

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Communications</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1038/s41467-023-37691-7</span>
      </div>

      
        
          <div class="citation-description">
            This study maps high-resolution kidney eQTLs using glomerular and tubulointerstitial biopsy samples integrated with single-nucleus open chromatin data. By applying Bayesian fine-mapping with an integrative prior, the work improves variant resolution, strengthens GWAS colocalization, and experimentally validates regulatory variants influencing kidney traits.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.nature.com/articles/s41467-023-37691-7" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93834" class="citation-image" aria-label="Information about immune cell proportions and tumor stage improves the prediction of recurrence in patients with colorectal cancer">
        <img src="" alt="Information about immune cell proportions and tumor stage improves the prediction of recurrence in patients with colorectal cancer" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93834" class="citation-title">
        Information about immune cell proportions and tumor stage improves the prediction of recurrence in patients with colorectal cancer
      </a>

      <div class="citation-authors" data-tooltip="JungHo Kong, Jinho Kim, Donghyo Kim, Kwanghwan Lee, Juhun Lee, Seong Kyu Han, Inhae Kim, Seongsu Lim, Minhyuk Park, Seungho Shin, Woo Yong Lee, Seong Hyeon Yun, Hee Cheol Kim, Hye Kyung Hong, Yong Beom Cho, Donghyun Park, Sanguk Kim" tabindex="0">
        JungHo Kong, Jinho Kim, Donghyo Kim, Kwanghwan Lee, Juhun Lee, …, Hee Cheol Kim, Hye Kyung Hong, Yong Beom Cho, Donghyun Park, Sanguk Kim

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Patterns</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.patter.2023.100736</span>
      </div>

      
        
          <div class="citation-description">
            This study presents a method that integrates immune cell proportion data with clinical information to improve recurrence risk prediction in colorectal cancer. Validated in two independent patient cohorts of different races, the approach enhances predictive accuracy and highlights potential therapeutic targets linked to immune cell dynamics.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.sciencedirect.com/science/article/pii/S2666389923000764?via%3Dihub" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93833" class="citation-image" aria-label="Tissue-specific and tissue-agnostic effects of genome sequence variation modulating blood pressure">
        <img src="" alt="Tissue-specific and tissue-agnostic effects of genome sequence variation modulating blood pressure" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93833" class="citation-title">
        Tissue-specific and tissue-agnostic effects of genome sequence variation modulating blood pressure
      </a>

      <div class="citation-authors" tabindex="0">
        Dongwon Lee, Seong Kyu Han, Or Yaacov, Hanna Berk-Rauch, Prabhu Mathiyalagan, Santhi K. Ganesh, Aravinda Chakravarti

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cell Reports</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.celrep.2023.113351</span>
      </div>

      
        
          <div class="citation-description">
            This study integrates GWAS, multiomics data, and machine learning to dissect the regulatory basis of blood pressure heritability. It identifies ~500,000 predicted regulatory variants across kidney, adrenal, heart, and artery tissues, showing that enhancer variants in tissue-relevant regulatory networks are major drivers of interindividual blood pressure variation.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01363-3" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g93832" class="citation-image" aria-label="OASIS portable: User-friendly offline suite for secure survival analysis">
        <img src="" alt="OASIS portable: User-friendly offline suite for secure survival analysis" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g93832" class="citation-title">
        OASIS portable: User-friendly offline suite for secure survival analysis
      </a>

      <div class="citation-authors" tabindex="0">
        Seong Kyu Han, Hyunwoo C. Kwon, Jae-Seong Yang, Sanguk Kim, Seung-Jae V. Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Molecules and Cells</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1016/j.mocell.2024.100011</span>
      </div>

      
        
          <div class="citation-description">
            OASIS portable is an offline, user-friendly suite for secure survival analysis, eliminating the need to upload data to online servers. It provides a virtualized version of the OASIS 2 webserver on personal computers, ensuring both accessibility and data security without requiring internet access.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://www.sciencedirect.com/science/article/pii/S1016847824000049" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g9383z" class="citation-image" aria-label="T scp CLASS scp : An Online Tool for the Identification and Classification of Aging and Senescence Using Transcriptome Data">
        <img src="" alt="T scp CLASS scp : An Online Tool for the Identification and Classification of Aging and Senescence Using Transcriptome Data" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g9383z" class="citation-title">
        T‐<scp>CLASS</scp>: An Online Tool for the Identification and Classification of Aging and Senescence Using Transcriptome Data
      </a>

      <div class="citation-authors" tabindex="0">
        Seung‐Chul J. Lee, Gee‐Yoon Lee, Sieun S. Kim, Yunkyu Bae, Seokjin Ham, Jooyeon Sohn, Seong Kyu Han, Seung‐Jae V. Lee

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Aging Cell</span>
         · 
        <span class="citation-date">[no date info]</span>
         · 
        <span class="citation-id">10.1111/acel.70193</span>
      </div>

      
        
          <div class="citation-description">
            T-CLASS is an online tool that identifies gene sets from transcriptome data to optimally represent longevity and aging paradigms. It shows robust classification performance across multiple datasets and successfully detects transcriptomic changes induced by lifespan-extending molecules, with experimental validation for rifampicin and atracurium.

          </div>
        

        
          <div class="citation-buttons">
            
              



  <div class="button-wrapper">
    <a class="button" href="https://onlinelibrary.wiley.com/doi/10.1111/acel.70193" data-tooltip="Website" data-style="bare" aria-label="Website">
      <i class="icon fa-solid fa-globe"></i>
      
        <span>Journal site</span>
      
    </a>
  </div>


            
          </div>
        

        
      
    </div>
  </div>
</div>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/background.jpg')" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->

  <div>
    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="mailto:skhan@inha.ac.kr" data-tooltip="Email" data-style="bare" aria-label="Email">
      <i class="icon fa-solid fa-envelope"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://orcid.org/0000-0001-9312-2864" data-tooltip="ORCID" data-style="bare" aria-label="ORCID">
      <i class="icon fa-brands fa-orcid"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://scholar.google.com/citations?user=SUkqU2cAAAAJ" data-tooltip="Google Scholar" data-style="bare" aria-label="Google Scholar">
      <i class="icon fa-brands fa-google"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://github.com/compgenai" data-tooltip="GitHub" data-style="bare" aria-label="GitHub">
      <i class="icon fa-brands fa-github"></i>
      
    </a>
  </div>


    
  </div>

  <div>
    © 2025
    Computational Genomics &amp; AI Lab
      |   Built with
    <a href="https://github.com/greenelab/lab-website-template">
      Lab Website Template
    </a>
  </div>

  <input type="checkbox" class="dark-toggle" data-tooltip="Dark mode" aria-label="toggle dark mode" oninput="onDarkToggleChange(event)">
</footer>

  </body>
</html>
